manifests/s/schollz/croc/9.6.2/schollz.croc.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: schollz.croc
+PackageVersion: 9.6.2
+InstallerType: zip
+ReleaseDate: 2022-12-06
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: croc.exe
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-64bit.zip
+  InstallerSha256: C46C2D6D2B92CCA3347FAB2D63544819BAAE9611460C67BAF78FFE399C7555AB
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: croc.exe
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-32bit.zip
+  InstallerSha256: 7CD7AB8681063B9CDA00A44E5FC57BBA9757911FC595C447286F90C6CAA4176A
+- Architecture: arm64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: croc.exe
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM64.zip
+  InstallerSha256: 9DB19070F24FA0BBD6DCA56F2CE72AD9941ECEA15215BAE967D35EC13B8B0FE8
+- Architecture: arm
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: croc.exe
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM.zip
+  InstallerSha256: 8D55CE05005928C75DC410C87453E3B2A0092632F314E45CF420FC0127DB3552
```

Comment:
```suggestion
NestedInstallerType: portable
NestedInstallerFiles:
- RelativeFilePath: croc.exe
  PortableCommandAlias: croc
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-64bit.zip
  InstallerSha256: C46C2D6D2B92CCA3347FAB2D63544819BAAE9611460C67BAF78FFE399C7555AB
- Architecture: x86
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-32bit.zip
  InstallerSha256: 7CD7AB8681063B9CDA00A44E5FC57BBA9757911FC595C447286F90C6CAA4176A
- Architecture: arm
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM.zip
  InstallerSha256: 8D55CE05005928C75DC410C87453E3B2A0092632F314E45CF420FC0127DB3552
- Architecture: arm64
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM64.zip
  InstallerSha256: 9DB19070F24FA0BBD6DCA56F2CE72AD9941ECEA15215BAE967D35EC13B8B0FE8
```

Reply:
YamlCreate doesn't seems to do this, would be a great feature.